src/types.rs

```diff
@@ -330,12 +332,13 @@ impl Default for RuntimeConfig {
 		RuntimeConfig {
 			http_server_host: "127.0.0.1".to_owned(),
 			http_server_port: 7000,
+			http_data_port: 9000,
```

Comment:
Why do we need different ports? Can't we serve on the same port (7000) by mapping to different URLs?

Reply:
Well I tried but it was not working for some reason! So I just created new ports. I will dig deeper into this